l10n_it_sdi_channel/models/sdi.py

```diff
@@ -165,7 +165,7 @@ def receive_fe(
         for file_name, file_content in file_name_content_dict.items():
             attachments_values = self._process_single_fe(
                 file_name,
-                file_content,
+                base64.decodebytes(file_content),
```

Comment:
Domanda: ha senso fare in modo che sia il chiamante del metodo `receive_fe` a fare la decode?

In fondo, `sdi_channel` è generico come modulo, e a priori non tutti i canali necessariamente prevedono base64.  L'idea di avere un canale che scarica le fatture e - prima di passarle alla `receive_fe` - sia costretto a encodarle base64 la trovo convoluta.

La `receive_fe()` dovrebbe accettare dei file XML, oltre che a degli zip, e immagino che anche i metadati siano testo... - i chiamanti passano dati binari, o zipfile, o `bytes` adatti al parsing diretto XML (ossia potenzialmente adatti per `xml.etree.ElementTree.fromstring()` - salvo errori di sintassi ovviamente). 

A proposito, poco sopra, riga 115, io invece di testare se il filename termina per .zip, che fa molto PC DOS,  farei una `try:` direttamente prima della `with Zipfile.zipfile(...)`, che fa molto python, o perlomeno userei la `zipfile.is_zipfile()`, che guarda il magic, se vogliamo mantenere l'`if`.